eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -624,6 +627,117 @@ votingSpec =
           -- The same vote should now successfully ratify the proposal
           passEpoch
           getLastEnactedCommittee `shouldReturn` SJust (GovPurposeId addCCGaid)
+        describe "Proposal deposits contribute to active voting stake" $ do
+          it "Directly" $ do
+            -- Only modify the applicable thresholds
+            modifyPParams $ \pp ->
+              pp
+                & ppDRepVotingThresholdsL
+                  .~ def
+                    { dvtCommitteeNormal = 51 %! 100
+                    , dvtCommitteeNoConfidence = 51 %! 100
+                    }
+                & ppCommitteeMaxTermLengthL .~ EpochInterval 20
```

Comment:
There is no need to modify these two , `ppDRepVotingThresholdsL` and  `ppCommitteeMaxTermLengthL`, because they are set in this way when initializing Conway

Reply:
Thanks for pointing this out :+1: